src/main/java/com/kwezal/bearinmind/core/auth/service/AuthService.java

```diff
@@ -56,21 +59,33 @@ public LoginResponseDto logIn(
             authClient,
             user.getId(),
             userCredentials.getUsername(),
+            user.getFullName(),
             userCredentials.getRole().getAuthorityNames(),
-            user.getLocale()
+            user.getLocale(),
+            user.getImage()
         );
     }
 
     @Transactional(readOnly = false)
     public LoginResponseDto signUp(final HttpServletResponse response, final CreateUserDto dto, final AuthClient authClient) {
         final var user = userService.createUser(dto);
+
+        StringJoiner fullNameJoiner = new StringJoiner(" ");
```

Comment:
Maybe should we add a fullName field to the UserDTO and move this method to the UserMapper to avoid duplicated code here and in the User entity?

Reply:
Makes sense, I'll move this code to user mapper.